crates/ruff_benchmark/benches/ty_walltime.rs

```diff
@@ -181,7 +181,7 @@ static PYDANTIC: Benchmark = Benchmark::new(
         max_dep_date: "2025-06-17",
         python_version: PythonVersion::PY39,
     },
-    3000,
+    5000,
```

Comment:
The increase here is caused by the fact that we now split diagnostic on unions into N_elements diagnostics.